projects/adrv9009zu11eg/common/adrv9009zu11eg_bd.tcl

```diff
@@ -158,25 +158,26 @@ ad_connect  sys_concat_intc_0/In0 GND
 # ADRV9009 Specific Connections
 # TX parameters
 
-set TX_NUM_OF_LANES 8      ; # L
-set TX_NUM_OF_CONVERTERS 8 ; # M
+set TX_NUM_OF_LANES $ad_project_params(JESD_TX_L); # L
+set TX_NUM_OF_CONVERTERS $ad_project_params(JESD_TX_M) ; # M
 set TX_SAMPLES_PER_FRAME 1 ; # S
 set TX_SAMPLE_WIDTH 16     ; # N/NP
 
 set TX_SAMPLES_PER_CHANNEL 2 ; # L * 32 / (M * N)
```

Comment:
If L and M is changing this should be recomputed too.

Same is valid for RX_SAMPLES_PER_CHANNEL and OBS_SAMPLES_PER_CHANNEL.